rsc/.gitignore

```diff
@@ -0,0 +1 @@
+/generalData.properties
```

Comment:
changes to the generalData.properties should not be ignored (now it is named settings.properties)

Reply:
I corrected